evap/staff/forms.py

```diff
@@ -60,9 +62,12 @@ def __init__(self, *args, **kwargs):
             self.fields['last_modified_user_2'].initial = self.instance.last_modified_user.full_name
         self.fields['last_modified_user_2'].widget.attrs['readonly'] = "True"
 
-        if self.instance.state == "inEvaluation":
+        if self.instance.state in ['inEvaluation', 'evaluated', 'reviewed']:
             self.fields['vote_start_date'].widget.attrs['readonly'] = "True"
-            self.fields['vote_end_date'].widget.attrs['readonly'] = "True"
+
+    def clean(self):
+        if self.cleaned_data['vote_end_date'] < datetime.date.today():
+            raise forms.ValidationError(_("The vote end date must be in the future."))
```

Comment:
was it possible previously to create courses which end date was not in the future?